packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -96,7 +96,7 @@ class CupertinoPageScaffold extends StatelessWidget {
             viewInsets: newViewInsets,
           ),
           child: Padding(
-            padding: EdgeInsets.only(top: topPadding, bottom: bottomPadding),
+            padding: EdgeInsets.only(bottom: bottomPadding),
```

Comment:
This should be in a separate PR - unless it actually is and this is part of the current mixup.